org.arrl.trustedqsl.json

```diff
@@ -0,0 +1,148 @@
+    "command" : "tqsl",
+    "finish-args" : [
+	"--share=network",
+	"--socket=x11",
+	"--filesystem=home"
+     ],
+    "modules" : [
+        {
+            "name" : "openssl",
+            "buildsystem" : "simple",
+            "build-commands" : [
+                "./Configure --prefix=/app && make depend && make build_libs && make install"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://www.openssl.org/source/openssl-3.1.0.tar.gz",
+                    "sha256" : "aaa925ad9828745c4cad9d9efeb273deca820f2cdcf2c3ac7d7c1212b7c497b4"
+                }
+            ],
+	    "cleanup" : [
+		"/bin/c_rehash",
+		"/bin/openssl",
+		"/lib64/libcrypto.a",
+		"/lib64/libssl.a",
+		"/share/doc/openssl/*",
+		"/share/doc/openssl",
+		"/share/man/man1/*",
+		"/share/man/man3/*",
+		"/share/man/man5/*",
+		"/share/man/man7/*",
+		"/share/man/man1",
+		"/share/man/man3",
+		"/share/man/man5",
+		"/share/man/man7",
+		"/include/openssl/*",
+		"/include/openssl",
+		"/lib64/pkgconfig/*",
+		"/lib64/pkgconfig"
+	    ]
+        },
+        {
+            "name" : "lmdb",
+            "buildsystem" : "simple",
+            "build-commands" : [
+                "cd libraries/liblmdb/ && make prefix=/app install"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://github.com/LMDB/lmdb/archive/LMDB_0.9.23.tar.gz",
+                    "sha256" : "abf42e91f046787ed642d9eb21812a5c473f3ba5854124484d16eadbe0aa9c81"
+                }
+            ],
+	    "cleanup" : [
+		"/include",
+		"*.a",
+		"/share/man",
+		"/bin/mdb_copy",
+		"/bin/mdb_dump",
+		"/bin/mdb_load",
+		"/bin/mdb_stat"
+	    ]
+        },
+        {
+            "name" : "curl",
+            "buildsystem" : "simple",
+            "build-commands" : [
+                "./configure prefix=/app --with-openssl && make install"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://curl.se/download/curl-8.0.1.tar.gz",
+                    "sha256" : "5fd29000a4089934f121eff456101f0a5d09e2a3e89da1d714adf06c4be887cb"
+                }
+            ],
+	    "cleanup" : [
+		"/bin/curl",
+		"/include/curl/*",
+		"/include/curl",
+		"/lib/libcurl.a",
+		"/bin/curl-config",
+		"/share/man/man1/*",
+		"/share/man/man3/*",
+		"/share/man/man7/*",
+		"/share/man/man1",
+		"/share/man/man3",
+		"/share/man/man7",
+		"/share/aclocal/libcurl.m4",
+		"/share/aclocal",
+		"/lib/pkgconfig/libcurl.pc",
+		"/lib/pkgconfig"
+	    ]
+        },
+	{
+	    "name" : "wxWidgets",
+	    "buildsystem" : "cmake-ninja",
+    	    "config-opts" : [
+		    "-DwxUSE_OPENGL=No",
+		    "-DwxUSE_HTML=Yes",
+		    "-DwxUSE_STREAMS=Yes",
+		    "-DwxUSE_WXHTML_HELP=Yes",
+		    "-DCMAKE_BUILD_TYPE=Release",
+		    "-DwxUSE_UNICODE=Yes"
+	    ],
+	    "sources" : [
```

Comment:
sources is usually last in the module by good practice. This applies everywhere.

Reply:
Corrected - moved the sources blocks to the end.